packages/backend/convex/users.ts

```
@@ -26,6 +26,8 @@ export const create = mutation({
       throw new Error('No organization');
     }
 
+    throw new Error('Not implemented');
+
     const id = await ctx.db.insert('users', args);
     return { id };
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)